ui/nuxeo-document-comments/nuxeo-document-comment-thread.js

```diff
@@ -75,17 +75,21 @@ import { FormatBehavior } from '../nuxeo-format-behavior.js';
               >
               </paper-textarea>
               <dom-if if="[[!_isBlank(text)]]">
-                <template>
+              <template>
+                <button class="ccomment-iron-icon" on-tap="_toggleDialog" aria-label$="[[label]]">
```

Comment:
mistake in class name `class="ccomment-iron-icon"`.
it should be `class="comment-iron-icon"`.